tripal_feature/theme/templates/tripal_feature_sequence.tpl.php

```diff
@@ -52,7 +52,32 @@
     $sequences_html .= '</div>';
     
   }
-  
+ 
+   // provide spliced mRNA sequences
+   // if this is an mRNA
+  if ($feature->type_id->name == 'mRNA') {
+
+
+      $seqs = tripal_get_feature_sequences(array('feature_id' => $feature->feature_id), array('derive_from_parent' => 1, 'aggregate' => 1,
+       sub_feature_types => array('exon') ,'is_html' => 1, 'width' => $num_bases));
+      //echo( ">". $seqs[0]['names']['header']. " \n".($seqs[0]['residues'])."\n" );
+      $list_items[] = '<a href="#mRNA">'. $feature->type_id->name . ' - spliced transcript sequence  ' . "</a>";
+      $sequences_html .= '<a name="mRNA"></a>';
+      $sequences_html .= '<div id="mRNA" class="tripal_feature-sequence-item">';
+      $sequences_html .= '<p><b>Spliced messenger RNA</b></p>';
+      $sequences_html .= '<pre class="tripal_feature-sequence">';
+      $sequences_html .= ">". $seqs[0]['defline']. " \n".$seqs[0]['residues'];
```

Comment:
What if there are no seqs?    Can we hide this whole display in that case?

Reply:
Fixed in commit 338342a1377463fbfa7d16baf793ed6fd8d0e433